thoth/storages/graph/janusgraph.py

```diff
@@ -1845,3 +1845,101 @@ def parse_python_solver_name(solver_name: str) -> dict:
 
         python_version = ".".join(list(python_version))
         return {"os_name": parts[0], "os_version": parts[1], "python_version": python_version}
+
+    def get_total_number_of_vertex_instances_count(self) -> int:
+        """Count the total number of instances of all edges in the graph database."""
+        query = (
+            self.g.V()
+            .has("__type__", "vertex")
+            .count()
+            .next()
+        )
```

Comment:
This one and the counting edge related queries will not hit any index I think. Do we want to have indeces for them?